src/sage/categories/finite_complex_reflection_groups.py

```diff
@@ -56,15 +56,15 @@ class FiniteComplexReflectionGroups(CategoryWithAxiom):
         sage: W = ComplexReflectionGroups().Finite().example(); W       # optional - gap3
```

Comment:
It seems the `# optional - gap3` could all go in the same column for the whole file except for this line?

Reply:
This one is at 72 (same as earlier up in the file). There are some occurrences around lines 900-1000 that I aligned at column 80 (rightmost place where the whole tag is visible) because the lines are longer. But in docstrings that have column-88 aligned tags, it has to be aligned farther to the left.